.circleci/config.yml

```diff
@@ -41,12 +41,20 @@ jobs:
     docker:
       - image: rust
       - image: ipfs/go-ipfs
+      - image: node
+        command: ["bash", "-c", "git clone https://github.com/libp2p/js-libp2p &&
```

Comment:
@kirushik Just in case, do you think there is a better way to do this?

Reply:
Well, let's create a proper image with those steps baked in, maybe?
That way, we won't have to run the same build steps on every test run.

Where and how to set that up — that's another question. Let's have a talk in person tomorrow.

Reply:
https://hub.docker.com/r/parity/libp2p-js-echo-server/